pkg/arguments/arguments.go

```diff
@@ -204,6 +206,14 @@ func CheckIgnoredAutoscalingFlags(autoscaling cluster.Autoscaling, computeNodes
 		if computeNodes != 0 {
 			return fmt.Errorf("--compute-nodes is meaningless with --enable-autoscaling")
 		}
+
+		if autoscaling.MinReplicas == 0 {
```

Comment:
Do we want to check `<= 0` just so that we can catch those before the API does?

Reply:
If we do, it's better to put in a validation callback, like `validateComputeNodes`, so that interactively it re-prompts if typed out of range value.  Although there is presently an annoying bug with re-prompting (prev text remains there but not visible).

Anyway, out of scope for this PR.

Reply:
BTW, how does auto-scaling work with multi-AZ?  Are the min/max per-AZ or * 3 for total?  Does backend reject non-multiples of 3?

(--compute-nodes currently takes total (multiple of 3), while UI takes per-AZ number.)

Reply:
@vkareh we validate min (which can be 2,3,4 or 9 according to multi-az and ccs booleans).

@cben autoscaling works the same with multi-az as compute nodes. Both min and max must be a multiple of 3.

Theres a validation callback for min that checks min number and multi az - `validateAutoscalingMin`
and a validator for max checking its >= min and multi az constraint - `validateAutoscalingMin`
Both sent to `promptInt`